src/main/groovy/io/openliberty/tools/gradle/utils/ArtifactDownloadUtil.groovy

```diff
@@ -36,6 +38,19 @@ public class ArtifactDownloadUtil {
 
         return downloadFile(project, config, coordinates)
     }
+	
+	public static File downloadSignature(Project project, String groupId, String artifactId, String type, String version, File esa) throws PluginExecutionException {
+		String coordinates = groupId + ":" + artifactId + ":" + version + "@" + type
```

Comment:
might be good to have a debug log statement in here to capture this copy of the signature file